crates/rolldown/tests/rolldown/issues/3438/artifacts.snap

```diff
@@ -75,5 +75,5 @@ const clear$1 = "repro3_clear$1";
 const clear = "repro3_clear";
 
 //#endregion
-export { clear$1 as clear$1$2, clear$2, clear as clear$3 };
+export { clear, clear$1, clear$2 };
```

Comment:
As you can see the new output is more intuitive since all export symbols in chunk `repro32.js` are not conflicted at all, they should keep it as is.